channels/layers.py

```diff
@@ -267,7 +266,8 @@ def _clean_expired(self):
         for group in self.groups:
             for channel in self.groups.get(group, set()):
                 # If join time is older than group_expiry end the group membership
-                if self.groups[group][channel] and int(self.groups[group][channel]) < (int(time.time()) - self.group_expiry):
+                if (self.groups[group][channel] and
+                    int(self.groups[group][channel]) < (int(time.time()) - self.group_expiry)):
                     del self.groups[group][channel]
```

Comment:
Maybe shortcut local variable will be better.

Reply:
I added a timeout variable